draft-tigress-requirements.md

```diff
@@ -167,8 +167,7 @@ A number of existing solutions / protocols have been reviewed in order to be use
 
 ## Arbitrary Messaging Channel (Email / WhatsApp / SMS / Signal / etc.)
 
-The Provisioning Information MAY be sent from Sender to Receiver over an arbitrary messaging channel, but that would not provide a good user experience. Users MAY need to copy and paste the Provisioning Information, or need a special application to handle some new file type. This violates (Req-SmoothUX).
-If multiple round trips were required the user would need to manually managing multiple payloads of Provisioning Information. This would be very hard for anyone non technical and greatly limit adoption. This violates (Req-NontechnicalUX).
+The Provisioning Information MAY be sent from Sender to Receiver over an arbitrary messaging channel that supports binary file transfer, but this would not support provisioning flows which require multiple round trips as requied by (Req-RoundTrips).
```

Comment:
Why couldn't the multiple round trips be performed over the arbitrary messaging channel? The receiver could send the reply over the messaging channel for the sender to handle. Then the sender could send the reply to be ingested by the receiver. Basically a certificate signing request or key signing request done over the messaging channel.

The only reasons I could think of that would preclude this flow are around user experience. Do you have an idea of other reasons that would preclude this?

Reply:
I can't think  of any other reasons and I'm wondering if we could solve it in a more elegant way. We can soften this to something like "this may not easily support" rather than "this would not support"

Reply:
I like the "this may not easily support" wording to soften the statement. 

If this came in a working group meeting would it be best to not mention the UX or practical application of the protocol? If so, it will be interesting to defend why we need this protocol.

Reply:
I think we need to justify when the protocol needs to handle multiple round trip provisioning flows, since it adds complexity. But once we have that requirement justified, arguments for the protocol to support it well for pragmatic reasons are entirely valid.

Reply:
Justification of multiple round trips lies in the format of encrypted provisioning information - simply put, Digital Car Key  (CCC spec) needs it, since Owner's signature is required over the Friend's device KeyImport request. 
But this will bring us into getting into provisioning phase and close ties with CCC spec, which we tried to avoid in the first place....

Reply:
We can possibly rephrase the sentence to  "If multiple round trips were required by the credential vertical that provisioning information belongs to (e.g. Digital Car Key - defined by CCC specification) ..."

Reply:
I believe, this is covered in #21

Reply:
I said in the PR #21 that we need to justify the assumption. I think we need to at least describe at a high level why the provisioning protocol requires multiple round trips.